index.rst

```diff
@@ -48,10 +48,63 @@
 
    **This technote is not yet published.**
 
-   This proposal outlines how observing tasks will be flowed from the planning stage through execution, then linked to data analysis technotes and/or notebooks.
+   This proposal outlines how observing tasks will be flowed from the planning stage through execution,
+   then linked to data analysis technotes and/or notebooks.
+
+Motivation for this proposal
+============================
+
+    * We want central repository for all information related to a task executed at night.
+    * We want to have a way to know how and when the tasks were run and how the data was processed.
+    * Search and find tasks and their results.
+
+Steps
+=====
+
+    * Run planning
+        * Which tasks/test are we running during a observation run?
+        * Do we want to store these tasks/tests as JIRA Tickets?
```

Comment:
I think you can just get rid of this question. We converged on using Jira, and we're using it in numerous other places throughout the project.